src/Microsoft.Data.SqlClient/tests/FunctionalTests/SqlExceptionTest.cs

```diff
@@ -33,7 +33,7 @@ public void SerializationTest()
             Assert.Equal(e.StackTrace, sqlEx.StackTrace);
         }
 
-#if !NET5_0
+#if !NET5_AND_ABOVE
```

Comment:
If it's not a big job could this be something like NET50_OR_LATER since the versions are continuous in time not number, that way if we somehow end up with a NetNext1.0 it's still later then net5 and the define isn't confusing, OR also makes it exclusive since you can't be both net5 and later than net5.

Reply:
done